src/main/java/uk/gov/justice/hmpps/prison/service/FinanceService.java

```diff
@@ -41,12 +47,34 @@
     private final AccountCodeRepository accountCodeRepository;
     private final OffenderSubAccountRepository offenderSubAccountRepository;
     private final OffenderTrustAccountRepository offenderTrustAccountRepository;
+    private final OffenderDamageObligationService offenderDamageObligationService;
 
     @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER", "GLOBAL_SEARCH"})
     public Account getBalances(final Long bookingId) {
-        return bookingRepository.getBookingAgency(bookingId)
-                .map(agency -> financeRepository.getBalances(bookingId, agency))
-                .orElse(null);
+
+        final var offenderSummary = bookingRepository.getLatestBookingByBookingId(bookingId)
+            .orElseThrow(new EntityNotFoundException("Booking not found for id: " + bookingId));
+
+        final var damageObligationBalance =
+            offenderDamageObligationService.getDamageObligations(offenderSummary.getOffenderNo(), "")
```

Comment:
Does this include only active damage obligations?

Reply:
done